code/modules/mob/living/say.dm

```diff
@@ -88,9 +90,12 @@ proc/get_radio_key_from_channel(var/channel)
 	if(GREY in mutations)
 		message = "<span class='grey'>[message]</span>"
 
-	else if(COMIC in mutations)
+	if(COMIC in mutations)

```

Comment:
Functionally, I don't see the point to it--regardless of `if` or `else if`, the comic mutation is going to override the grey mutation if someone ultimately has both.